src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Controllers/AuthenticationController.cs

```diff
@@ -800,6 +800,11 @@ private static UserAuthenticationModel GetUserFromToken(JwtSecurityToken jwtSecu
                 }
             }
 
+            if (userAuthenticationModel.AuthenticationMethod.Equals(AuthenticationMethod.NotDefined))
+            {
+                userAuthenticationModel.AuthenticationMethod = (Enum.AuthenticationMethod)System.Enum.Parse(typeof(Enum.AuthenticationMethod), provider.DefaultAuthenticationMethod);
```

Comment:
Unnecessary to include namespaces everywhere.
```suggestion
                userAuthenticationModel.AuthenticationMethod = (AuthenticationMethod)AuthenticationMethod.Parse(typeof(AuthenticationMethod), provider.DefaultAuthenticationMethod);
```